templates/page/page.tpl.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Interesting. I've generally preferred to keep them visible at a simple `ls` glance, and I don't know whether we'll end up having very many here, but go for it. As long as we stick with the [naming conventions](https://www.drupal.org/node/1089656) all should be well.